google_api_availability/pubspec.yaml

```diff
@@ -10,16 +10,18 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  meta: ^1.8.0
+  google_api_availability_android: ^1.0.0+1
+  google_api_availability_platform_interface: ^1.0.0
```

Comment:
You can also upgrade this version since I merged the other one, Maurits will have to release all of them though.